File: ProjectEuler/p1.cpp
// Multiples of 3 and 5
#include <bits/stdc++.h>

using namespace std;

int main() {
    int sum;
    for (int i = 0; i < 1000; i++) {
        if ((i % 3 == 0) || (i % 5 == 0)) {
            sum += i;
        }
    }
    cout << sum << "\n";
    return 0;
}

File: Project Euler/p1.py
# Multiples of 3 and 5
def main():
    sum = 0
    for i in range(1000):
        if i % 3 == 0 or i % 5 == 0:
            sum += i
    return sum

if __name__ == "__main__":
    print(main())
